src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -1571,6 +1571,7 @@
                     <ref name="inlinenoprodnote"/> <!-- em, strong, dfn, code, samp, kbd, span, abbr, math, a, img, br, q, sub, sup, bdo -->
                     <ref name="lic"/> <!-- span -->
                     <ref name="list"/> <!-- ol, ul -->
+                    <ref name="pagebreak.block"/> <!-- div -->
```

Comment:
I think this should be `pagebreak.inline`. The `<choice>` here chooses between inline content (the first `<choice>`) and block content (the second `<choice>`). The reference to `list` here allows for how navigation documents are marked up.

Reply:
Hi @josteinaj 

This will solve the issue in question; a block page break was used inside of a <LI> before the definition of another <OL> element. The inline page break is used mostly inside of paragraphs. Perhaps it's allowed here inside the inlinenoprodnote rule.

Best regards
Daniel

Reply:
Perhaps @martinpub or @AndersEkl could shed some light here. Do we allow block page breaks inside of list items?

Reply:
It depends on the context. If the list item only has text inside it, with no p or other block elements, only inline page breaks would be allowed.

But if the list items contains other block elements, such as paragraphs or nested lists, then a block page break between other block elements would be ok, I'd say.

Reply:
Hi @josteinaj 

Did this issue resolve itself, or do you still want me to change something?

Best regards
Daniel

Reply:
I've now changed this to inline.